src/ophys_etl/modules/segmentation/detect/feature_vector_utils.py

```diff
@@ -0,0 +1,246 @@
+from typing import Tuple, Optional, List
+import numpy as np
+
+
+def choose_extreme_pixels(
+        image_data: np.ndarray,
+        delta_z: List[float],
+        pixel_ignore: Optional[np.ndarray] = None) -> List[Tuple[int, int]]:
+    """
+    For a specified list of values delta_z, choose the pixels in an
+    image that are that many standard deviations from the maximum
+    and minimum flux values
+
+    i.e. if delta_z = [1, 2]
+
+    The output will be the row, column coordinates of the pixels
```

Comment:
I assume this is "the pixels closest to" these flux values, as there is no guarantee there would be pixels with the exact flux values.
I think this docstring could be more clear.

Reply:
I've edited that line of the docstring so that it claims only to return pixels closest to the target values of flux